tests/maneuver_test.py

```diff
@@ -0,0 +1,51 @@
+from queue import PriorityQueue
+import pytest
+from time import time
+
+from main import MainSatelliteThread
+from flight_modes.maneuver_flightmode import ManeuverMode
+import utils.constants as consts
+import utils.parameters as params
+
+# to speed testing
+params.GLOW_WAIT_TIME = 0.01
+
+
+class FakeGOM:
+    def glowplug(self, x):
+        return 0
+
+    def glowplug2(self, x):
+        return 0
```

Comment:
Really jank, but this worked on my machine:
```suggestion
class FakeGOM(Gomspace):

    def __init__(self):
        self.pc = None

    def glowplug(self, duration, delay=0):
        return None

    def glowplug2(self, duration, delay=0):
        return None
```